README.md

```diff
@@ -2,29 +2,36 @@
 
 Hybrid Application Console Infrastructure repository
 
-## Local Development
+## Run with webpack proxy
 
-Since the plugin is not served directly from console.redhat.com you have to run webpack proxy in order to properly develop locally. You have two options
+Since the plugin is not served directly from console.redhat.com you have to run webpack proxy in order to properly develop locally.
 
-* With hac-core served from CDN - if you want to test your changes just in your plugin
-* With hac-core locally - if you want to add new plugin or change how navigation is treated.
+### Update `/etc/hosts`
 
-### With hac-core served from CDN (default)
-
-Once your plugin has been enabled in hac-core and promoted to at least stage environment you can start serving hac-core from CDN. You can just run
+To use webpack proxy you need to append this to your `/etc/hosts` for auth:
 
 ```
-yarn dev
+127.0.0.1 prod.foo.redhat.com
+127.0.0.1 stage.foo.redhat.com
+127.0.0.1 qa.foo.redhat.com
+127.0.0.1 ci.foo.redhat.com
```

Comment:
These do not exist anymore -- @karelhala please confirm

Reply:
Confirmed with @karelhala that we shouldn't be using these. I've updated the Readme.